code/game/objects/effects/anomalies/anomalies_bioscrambler.dm

```diff
@@ -0,0 +1,89 @@
+/obj/effect/anomaly/bioscrambler
+	name = "bioscrambler anomaly"
+	icon_state = "bioscrambler"
+	anomaly_core = /obj/item/assembly/signaler/anomaly/bioscrambler
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
+	layer = ABOVE_MOB_LAYER
+	lifespan = ANOMALY_COUNTDOWN_TIMER * 2
+
+	/// Who are we moving towards?
+	var/datum/weakref/pursuit_target
+	/// Cooldown for every anomaly pulse
+	COOLDOWN_DECLARE(pulse_cooldown)
+	/// How many seconds between each anomaly pulse
+	var/pulse_delay = 10 SECONDS
+	/// Range of anomaly pulse
+	var/range = 2
+
+/obj/effect/anomaly/bioscrambler/Initialize(mapload, new_lifespan, drops_core)
+	. = ..()
+	pursuit_target = WEAKREF(find_nearest_target())
+
+/obj/effect/anomaly/bioscrambler/anomalyEffect(seconds_per_tick)
+	. = ..()
+	if(!COOLDOWN_FINISHED(src, pulse_cooldown))
+		return
+
+	new /obj/effect/temp_visual/circle_wave/bioscrambler(get_turf(src))
+	playsound(src, 'sound/effects/cosmic_energy.ogg', vol = 50, vary = TRUE)
+	COOLDOWN_START(src, pulse_cooldown, pulse_delay)
+	for(var/mob/living/carbon/human/nearby in viewers(range, src))
+		nearby.getarmor(type = BIOACID)
```

Comment:
What do we get the armor here for? Maybe do a check here for anomaly susceptiability like artifacts already do?